open-company/open-company-web
# Notification light theme

Card: https://trello.com/c/TwWjv2jC

To test:
- open carrot
- open console
- run: `oc.web.actions.notifications.show_notification(cljs.core.merge(oc.web.lib.utils.app_update_error, clojure.walk.keywordize_keys(cljs.core.js__GT_clj({"expire": 0, "title": "Test", "description": "Test here too", "id": cljs.core.keyword("hello-there")}))))`
- before dismissing the notification
- open the menu
- [x] do you see the notification turning light on dark? Good
- close the menu
- [x] do you see it turning back to dark on light? Good
- [x] check it stays light when you open reminders
- [x] check it stays light when you open org settings
- [x] check it stays light when you open user settings
- [x] check it stays light when you open reminders
- [x] check it turns light when you open section settings
- [x] check it turns light when you open add a section
- now dismiss the notification
- go to user settings
- change your avatar
- [x] notification is light? Good
- [x] repeat with org icon
